### PR TITLE
chore: update annotation metadata panel

### PR DESCRIPTION
**Pull request by kne42**
#434, #388
- change publication order
- 'Publications' -> 'Publication'
- add method type field (hidden without param `enable-feature=methodType`)
- add annotation id
- hide author affiliation names
- update annotation author styling
- fix in-table styling for common authors component
- hide curator recommended field
- fix duplicate last modified field
- fix styling for ground truth, precision, and recall fields

Screenshot 2024-03-13 at 12 03 18 PM

### CODE REVIEW

**Review by Janeece — APPROVED** [review]
LGTM!

@kne42 One thing i want to confirm -- in your example screenshot, the metadata for "annotation method" says "manual" & "method type" is has no value. Am I correct in interpreting this as:
- Method type hasn't been assigned by Utz yet
- Annotation Method as "manual" is what was in the metadata, and it's just a coincidence that its the same as one of our categories

If my interpretation is correct then we are all good to go for me. (I just want to be sure I wasn't assuming my interpretation is correct and miss a bug)

File: frontend/packages/data-portal/app/components/Run/AnnotationConfidenceTable.tsx [hunk 1]
```diff
@@ -21,17 +21,19 @@ export function AnnotationConfidenceTable() {
...
         },
-        {
-          label: t('curatorRecommended'),
-          values: [annotation.is_curator_recommended ? t('yes') : '--'],
-        },
+        // Disabled as per design
+        // {
+        //   label: t('curatorRecommended'),
+        //   values: [annotation.is_curator_recommended ? t('yes') : '--'],
+        // },
```
[the remark this answers is not in the document]
Comment (kne42) [reply frontend/packages/data-portal/app/components/Run/AnnotationConfidenceTable.tsx:28]: good point, I guess I was just afraid it'll get re-added at some point 😅

File: frontend/packages/data-portal/app/components/Run/AnnotationOveriewTable.tsx [hunk 2]
```diff
@@ -12,50 +33,64 @@ export function AnnotationOverviewTable() {
...
+        // {
+        //   label: t('affiliationNames'),
+        //   values: Array.from(
+        //     new Set(
+        //       annotation.author_affiliations
+        //         .flatMap((author) => author.affiliation_name)
+        //         .filter((value): value is string => !!value),
+        //     ),
+        //   ),
+        // },
```
[the remark this answers is not in the document]
Comment (kne42) [reply frontend/packages/data-portal/app/components/Run/AnnotationOveriewTable.tsx:87]: it's not present in the design so I'd say yes

[the remark this answers is not in the document]
Comment (kne42) [reply frontend/packages/data-portal/app/components/Run/AnnotationOveriewTable.tsx:87]: https://www.figma.com/file/AELEVptBVyqp0vvFrzgfK6/4---Single-Run?node-id=14%3A17&mode=dev

**Comment by kne42** [comment]
@Janeece yes, everything you said is correct

**Pull request merged**